### DISCUSSION TITLE
How to turn of nested fields setting?

### DISCUSSION THREAD

**Discussion by SMKH-PRO** [react-hook-form/react-hook-form#9298] [Q&A]
Hi,

Due to a certain conditions we've to set keys with names like "aaa.1a", meaning that the names contains dot "." but react-hook-form is considering it  as nested object instead of a key.

I know there are a-lot of alternative ways to resolve this problem such as conversion of keys after and before putting them to react-hook-form, but I believe there's builtin feature to turn off nesting support that would be a-lot better than any alternative solution

Is there a way that we can turn off nested value setting?

Example:

Form Object:
```
{
 "aaa.1a":"expectedValue"
}
```
now if i try to set this field using setvalue.

setValue("aaa.1a", "expectedValue" )
the output of this will be 
```
{
   aaa:{
      "1a": "expectedValue"
     }
}
```

**Comment by bluebill1049** [comment]
'aaa-1a' is an option for you?
